src/Compilers/Core/Portable/System/Reflection/Metadata/ILOpCode.cs

```diff
@@ -204,6 +204,9 @@ enum ILOpCode : ushort
         Leave_s = 0xde,
         Stind_i = 0xdf,
         Conv_u = 0xe0,
+        // LoadMethodTokenAsInteger_i4 is a pseudo-opcode used to trick the metadata writer into putting a method token value into a ldc_i4.
```

Comment:
- When metadata writer separates from Roslyn and becomes a part of System.Metadata, is this going to be a documented and supported behavior?
- Wheat happens if the IL is subsequently rewritten and token change? 
- Is this supposed to work with .Net native?